src/Fixer/CastNotation/ModernizeTypesCastingFixer.php

```diff
@@ -91,9 +91,17 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                     continue;
                 }
 
+                $endIndex = $closeParenthesis;
+                $prevIndex = $tokens->getPrevMeaningfulToken($endIndex);
+                if ($tokens[$prevIndex]->equals(',')) {
+                    $tokens->removeTrailingWhitespace($prevIndex);
```

Comment:
As edge case, people can place space before the comma as well =)

Reply:
Do you mean like in the 2nd test case I've added?

Reply:
Yep, seems you already covered it =)